### PR TITLE
[Internal] FaultInjection: Refactors FaultInjection Tests

### PR DESCRIPTION
**Pull request by NaluTripician**
# Pull Request Template

## Description

Refactors tests for clarity. No behavior change.

## Type of change

Please delete options that are not relevant.

- [] Bug fix (non-breaking change which fixes an issue)

## Closing issues

To automatically close an issue: closes #IssueNumber

### CODE REVIEW

**Review by kirankumarkolli — APPROVED** [review]

File: Microsoft.Azure.Cosmos/FaultInjection/tests/Utils/TestCommon.cs [hunk 1]
```diff
@@ -18,6 +18,11 @@ internal static class TestCommon
 
         public const string EndpointMultiRegion = "";
         public const string AuthKeyMultiRegion = "";
+
+        internal static string GetConnectionString()
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos/FaultInjection/tests/Utils/TestCommon.cs:22]: Is this change related to refractoring?

**Review by kundadebdatta — APPROVED** [review]
LGTM.
